File: Sem1/Assignment/code/LBEP_Ex3.cpp
#include <iostream>

int main(){
    double x;
    std::cout << "Enter the radius of Circle: ";
    std::cin >> x;
    std::cout << "\tPerimter of the Circle: 2 * "<< x<<" * 3.14 = "<<x*2*3.14<<"\n";
    std::cout << "\tArea of the Circle: "<<x <<" * "<<x<<" *3.14 = "<<x*x*3.14<<"\n";
    return 0;
}

File: Sem1/Assignment/code/LBEP_Ex3.cs
using System;
class Program{
    static void Main(String[] agrs){
        Console.WriteLine("Enter the radius of Circle: ");
        string str = Console.ReadLine();
        double num = Double.Parse(str);
        Console.WriteLine("\tPerimeter of the Circle: 2 * {0:F2} * 3.14 = {1:F2}\n",num,num*2*3.14);
        Console.WriteLine("\tArea of the Circle: {0:F2} * {0:F2} * 3.14 = {1:F2}\n",num, num*num*3.14);
    }
}